src/dawn/Optimizer/PassStageSplitter.cpp

```diff
@@ -110,6 +166,45 @@ bool PassStageSplitter::run(const std::shared_ptr<StencilInstantiation>& stencil
           stage.update();
           ++stageIt;
         }
+
+      };
+}
+} // anonymous namespace
+
+bool PassStageSplitter::run(const std::shared_ptr<StencilInstantiation>& stencilInstantiation) {
```

Comment:
as discussed, maybe we can have the debug thing only for the mss, since it will already serve the purpose